_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,14 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused `mouseover` events to focus buttons and prevent keyboard navigation. 
+    summaryAdditional:  Now, hovering the mouse over date picker buttons will only cause the hover state.
```

Comment:
**suggestion (non-blocking):** Suggestion for simpler language for readability. 

```suggestion
    summaryAdditional:  When you hover your mouse over the date picker buttons, only the hover state will be triggered.
```

Reply:
Resolved in [9b7624d](https://github.com/uswds/uswds-site/pull/2496/commits/9b7624d09fda30b83af6b87f0330f00dafb20866)